ubbagent.Makefile

```diff
@@ -11,7 +11,8 @@ include $(makefile_dir)/common.Makefile
 	mkdir -p "$@"
 
 
-.build/ubbagent/ubbagent: | .build/ubbagent
+.build/ubbagent/ubbagent: $(shell find "$(MARKETPLACE_TOOLS_PATH)/vendor/ubbagent" -type f ) \
```

Comment:
does this slow `make`?...

Reply:
If it did, I could not tell the difference.